src/migrations/20240807162056-removeExtrinsicHash.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.removeColumn('CTypes', 'extrinsicHash');
+  },
+
+  async down(queryInterface, Sequelize) {
+    await queryInterface.addColumn('CTypes', 'extrinsicHash', {
+      type: Sequelize.STRING,
+      allowNull: false,
+    });
+  },
```

Comment:
Could you please test this?

Reply:
it will be tested on peregrine

Reply:
The "down" part will not be tested on peregrine

Reply:
I will try to do it locally. But my database does not have `extrinsicHash`.

Maybe I have to get one first by using `main` first.

Reply:
ok, on it

Reply:
```
 npx sequelize-cli db:migrate:undo                                       

Sequelize CLI [Node: 20.15.0, CLI: 6.6.2, ORM: 6.37.3]

Parsed url postgres://postgres:*****@localhost:5432/postgres
No executed migrations found.
```

Can't `undo` what has never been done.

Reply:
I'm getting a legacy database thanks to the `main` branch now. Then I should be able to properly test this. 

I'm doing it on spiritnet, because subscan is not available for peregrine. 
Descriptions and tags will not be available to test then. because I would need to create new ctypes

Reply:
I notice the errors, learn a couple of new things and fixed now. 

The `search` field needs to be dropped and redefined each time it's dependencies change.

Reply:
Wow, I have not even considered the search field here. See how helpful actual testing is? :) 

The thing I wanted to point out was the following. When you restore the extrinsicHash column and it’s non-nullable but without the default value, would not the database complain?

Reply:
Very helpful.

I just added the missing default for `extrinsicHashes` created during this down migration.